webapp/src/components/expanded_view/global_banner.tsx

```diff
@@ -29,55 +38,74 @@ export default function GlobalBanner(props: Props) {
                 <CompassIcon icon={props.icon}/>
             </Icon>
             <Body>{props.body}</Body>
-            { props.onClose &&
-            <CloseButton
-                className='style--none'
-                onClick={props.onClose}
-            >
-                <CompassIcon icon='close'/>
-            </CloseButton>
+            {props.onClose &&
+                <OverlayTrigger
+                    placement='left'
+                    key={'dismiss-banner'}
+                    overlay={
+                        <Tooltip id='dismiss-banner'>
+                            {formatMessage({defaultMessage: 'Dismiss'})}
+                        </Tooltip>
+                    }
+                >
+                    <CloseButton
+                        $bgHover={hoverMap[props.type]}
+                        className='style--none'
+                        onClick={props.onClose}
+                    >
+                        <CompassIcon icon='close'/>
+                    </CloseButton>
+                </OverlayTrigger>
             }
-            { !props.onClose &&
-            <RightFiller/>
+            {!props.onClose &&
+                <RightFiller/>
             }
         </Banner>
     );
 }
 
-const Banner = styled.div<{$color: string, $bgColor: string}>`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  width: 100%;
-  height: 40px;
-  background-color: ${({$bgColor}) => $bgColor};
-  font-size: 14px;
-  color: ${({$color}) => $color};
-
-  a, a:hover, a:visited {
+const Banner = styled.div<{ $color: string, $bgColor: string }>`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    background-color: ${({$bgColor}) => $bgColor};
+    font-size: 14px;
+    gap: 4px;
     color: ${({$color}) => $color};
-  }
 
-  i {
-    font-size: 18px;
```

Comment:
This should stay - the icon font-size should still be 18px

Reply:
@matthewbirtch It is still there, just moved to where it should be.

Reply:
Hmmm, it doesn't seem like it's getting applied then. I see you have added it on the close button, but this icon font-size should apply in the banner area as well

Reply:
Yep you're right, my mistake.